backends/ebpf/ebpfControl.cpp

```diff
@@ -47,21 +47,65 @@ bool ControlBodyTranslator::preorder(const IR::PathExpression* expression) {
 }
 
 void ControlBodyTranslator::processFunction(const P4::ExternFunction* function) {
-    ::error("%1%: Not supported", function->method);
+    if (function->method->name.name == "assert") {
+        BUG_CHECK(function->expr->arguments->size() == 1,
+                "%1%: Expected one argument in ", function);
+        builder->emitIndent();
+        builder->append("if (");
+        visit(function->expr->arguments->at(0)->expression);
+        builder->append(" == false) ");
+        builder->blockStart();
+        builder->emitIndent();
+        builder->appendFormat("% s = % s;",
```

Comment:
You chose a particular semantics for assert which is not the standard C one.
You should probably document this in a summary of the PR.
This should also go to the language design working group.